yt/yt/server/lib/chunk_pools/ordered_chunk_pool.cpp

```diff
@@ -441,7 +442,8 @@ class TOrderedChunkPool
 
         // Unbounded dynamic store cannot be split.
         if (dataSlice->GetSingleUnversionedChunkSlice()->GetInputChunk()->IsOrderedDynamicStore() &&
-            !dataSlice->GetSingleUnversionedChunkSlice()->UpperLimit().RowIndex) {
+            !dataSlice->GetSingleUnversionedChunkSlice()->UpperLimit().RowIndex)
```

Comment:
IMHO
It is better to separate stylistic and substantive changes into different pull requests.